### ISSUE TITLE
Cannot get autoheight for Server Side rows

### ISSUE THREAD

**Issue by CROSP** [ag-grid/ag-grid#3244]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of ag-Grid'
[] support request => see 'Requesting Community Support'
```
I am trying to get the auto height feature to work with the Server Side table model, however it still not wrapping the content of the rows. I've spent a lot of time trying to resolve the issue, however none of attempts succeed.
 
**Providing a Reproducible Scenario**

Here is my configuration:

```
    <ag-grid-vue
      class="ag-theme-material grid-table"
      :gridOptions="gridOptions"
      @grid-ready="onGridReady"
      :columnDefs="columnDefs"
      :defaultColDef="defaultColDef"
      :rowModelType="rowModelType"
      :domLayout="domLayout"
      :frameworkComponents="frameworkComponents"
      :sideBar="sideBar"
      :paginationPageSize="paginationPageSize"
      :animateRows="true"
      :floatingFilter="true"
      :serverSideDatasource="serverSideDatasource"
      :pagination="true">
    </ag-grid-vue>
```
Configuration for some related properties:

```
this.columnDefs = [
        ....
        {
          headerName: "Full name",
          cellClass: "cell-wrap-text",
          autoHeight: true,
          width: 150,
          field: "fullName"
        },
        {
          headerName: "Address",
          field: "address",
          sortable: false,
          width: 150,
          filter: false,
          cellClass: "cell-wrap-text",
          autoHeight: true,
          cellRenderer: "addressRenderer"
        },
        {
          headerName: "City",
          autoHeight: true,
          width: 120,
          cellClass: "cell-wrap-text",
          field: "city",
          filterParams: {
            suppressAndOrCondition: true
          },
          filter: "agTextColumnFilter"
        },
      ];
this.rowModelType = "serverSide";
this.rowGroupPanelShow = "never";
this.defaultColDef = {
  width: 100,
  filter: true,
  sortable: true,
  resizable: true
};
```

**Current behavior**

Text is not wrapped, but takes the single line

![image](https://user-images.githubusercontent.com/5072318/60687342-a4f0ed00-9eb6-11e9-87e0-1a6e1974c2e7.png)

**Expected behavior**

Should wrap text and set the row height accordingly

**Please tell us about your environment:**

```
     "vue": "^2.5.21",
    "ag-grid-community": "^21.0.1",
    "ag-grid-enterprise": "^21.0.1",
    "ag-grid-vue": "^21.0.1",

```
* **ag-Grid version:** 21.0.1

* **Browser:**
`"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36"`

* **Language:** [ES6/7 ]

**Comment by davis4acca** [comment]
Hi,

Thank you for your email. We are aware of this issue and have a feature request in our backlog:

You can follow the status of feature requests, bugs and releases using our pipeline: 
https://www.ag-grid.com/ag-grid-pipeline/

If you are an ag-Grid Enterprise Customer, please raise this in our Zendesk support system - contact info@ag-grid.com for access. This is our primary channel for Support.



This is the reference and summary of the relevant issue:

AG-3068	cellRenderer + autoHeight not working in the latest version

**Comment by CROSP** [comment]
@davis4acca Thank you so much for your detailed response. We are really missing this feature and it is the only issue that stops us from purchasing the Enterprise License. 
Will be waiting for the resolution. 

Thanks,
Have a great day

**Comment by CROSP** [comment]
Are there any updates on the issue?

**Comment by kiril-matev** [comment]
Just to follow up on this - the bug in that github issue was recorded as AG-3068 which was resolved in v21.1, released in Jul 2019, so that issue was resolved.

The best way to get an update on any bug is to look up the item using the reference provided above (AG-XXXX) in the changelog page:
https://www.ag-grid.com/ag-grid-changelog/

If the item appears in the changelog, it was fixed. If you can't find the item in the changelog, it means it's not been addressed yet.
